python/sglang/srt/models/phi3_small.py

```diff
@@ -0,0 +1,463 @@
+import math
+from typing import Dict, Iterable, List, Optional, Tuple, Union
+
+import torch
+from torch import nn
+from transformers import Phi3Config
+from transformers.configuration_utils import PretrainedConfig
+from vllm.distributed import get_pp_group, get_tensor_model_parallel_world_size
+from vllm.model_executor.layers.rotary_embedding import get_rope
+from vllm.model_executor.model_loader.weight_utils import default_weight_loader
+from vllm.model_executor.models.utils import (
+    make_empty_intermediate_tensors_factory,
+    make_layers,
+    maybe_prefix,
+)
+from vllm.sequence import IntermediateTensors
```

Comment:
This has not been supported. Can you delete everything related to `IntermediateTensors`?